### ISSUE TITLE
Update data-model for send-community in eos_cli_config_gen

### ISSUE THREAD

**Issue by onurgashi** [aristanetworks/ansible-avd#593]
**Issue Type**

- [x] Template enhancement
- [ ] Role enhancement
- [ ] Documentation enhancement (avd.sh)

**Is your feature request related to a problem? Please describe.**

As we have changed the data-model for send-community in the VRFs, it will allow greater flexibility to decide what type of community we would like to send out, instead of true/false.

Changes should happen in `router-bgp.j2`

This will introduce data-model change.

```
{%             if router_bgp.peer_groups[peer_group].send_community is defined and router_bgp.peer_groups[peer_group].send_community == true %}
   neighbor {{ peer_group }} send-community
{%             endif %}
```

**Describe the solution you'd like**

Select which community we want to send asides default (everything).

```
  send_community: < string | leave empty for all communities >
```

```
{%                      if router_bgp.vrfs[vrf].neighbors[neighbor].send_community is defined %}
      neighbor {{ neighbor }} send-community {{ router_bgp.vrfs[vrf].neighbors[neighbor].send_community }}
{%                      endif %}
```

**Describe alternatives you've considered**

**Additional context**

**Comment by ClausHolbechArista** [comment]
I believe some variant of this has been implemented recently, but it also matches on the string value "None".

**Comment by ClausHolbechArista** [comment]
Fixed / replaced by PR #812
